go/chat/search/indexer.go

```diff
@@ -470,27 +469,16 @@ func (idx *Indexer) allConvs(ctx context.Context, uid gregor1.UID, convID *chat1
 	return convMap, nil
 }
 
-func (idx *Indexer) convsByMTime(ctx context.Context, uid gregor1.UID,
+func (idx *Indexer) convsPrioritySorted(ctx context.Context, uid gregor1.UID,
 	convMap map[string]types.RemoteConversation) (res []types.RemoteConversation) {
 	res = make([]types.RemoteConversation, len(convMap))
 	index := 0
 	for _, conv := range convMap {
 		res[index] = conv
 		index++
 	}
-	_, ib, err := storage.NewInbox(idx.G()).ReadAll(ctx, uid, true)
```

Comment:
not sure why this was necessary, might have been remnant of an older version. Seemed like it could be nuked